Applications/LLMEval/ContentView.swift

```diff
@@ -6,20 +6,31 @@ import MLXRandom
 import Metal
 import SwiftUI
 import Tokenizers
+import MarkdownUI
 
 struct ContentView: View {
 
     @State var prompt = "compare python and swift"
     @State var llm = LLMEvaluator()
 
     var body: some View {
-        VStack {
-            // show the model output
-            ScrollView(.vertical) {
+        VStack (alignment: .leading) {
+            VStack {
+                HStack {
+                    Text(llm.modelInfo)
+                        .textFieldStyle(.roundedBorder)
+                    
+                    Spacer()
+                    
+                    Text(llm.stat)
+                }
                 if llm.running {
                     ProgressView()
                 }
-                Text(llm.output)
+            }
+            // show the model output
+            ScrollView(.vertical) {
+                Markdown(llm.output)
```

Comment:
Looks nice!  There was an outstanding PR that makes this scroll as text is appended and it causes a minor conflict here, see #24

Reply:
I added the changes from #24 to the ContentView. See: https://github.com/ml-explore/mlx-swift-examples/pull/27/commits/441263bbb1aeff441ce24c9b44261953e87e1c6a

Let me know if any additional changes would be helpful to resolve the conflict.

Reply:
I think you just need to rebase it on main, so something like this:

- update your fork of main
- update your local copy of main
    - `git checkout main`
    - `git pull`
- switch back to your branch and rebase it
    - `git checkout markdownUI`
    - `git rebase main`
    - resolve the conflicts
    - `git add` the files
    - `git rebase --continue` to finish the rebase
- force push your branch
    - `git push -f`

Something like that.  There are various guides if needed: https://docs.gitlab.com/ee/topics/git/git_rebase.html

Once that is done the commits will be on top of the current HEAD and the merge will be unblocked.

Reply:
Really appreciate the detailed step by step. I rebased and reformatted with pre-format as instructed.